geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -55,10 +56,11 @@ case class EmptyPlan(filter: HBaseFilterStrategyType) extends HBaseQueryPlan {
 case class ScanPlan(filter: HBaseFilterStrategyType,
                     table: TableName,
                     ranges: Seq[Scan],
+                    remoteFilters: Seq[HBaseFilter] = Nil,
```

Comment:
this needs to be added to HBaseQueryPlan, and then implemented in EmptyPlan and GetPlan